### ISSUE TITLE
Documentation schemas issue 

### ISSUE THREAD

**Issue by vtedesco** [schmittjoh/JMSPaymentCoreBundle#160]
The 3 schemas generated for this documentation page http://jmsyst.com/bundles/JMSPaymentCoreBundle/master/model look like wrong to me.

Sample :

![png](https://cloud.githubusercontent.com/assets/706825/5567366/aba35a98-8f43-11e4-81b6-13bf944dab67.png)

**Comment by regularjack** [comment]
I know it's been a while but could you please describe why you find the schemas to be wrong? What exactly is the problem? Thanks.

**Comment by vtedesco** [comment]
It's hard to read, arrow are pointing to nowhere because boxes are not properly aligned. I don't know about the data, I'm juste speaking about the visual rendering.

**Comment by regularjack** [comment]
I see what you mean, thanks for reporting it. I'm pretty sure the images were correct in the past, not sure what happened here. Will look into it.

**Comment by greg0ire** [comment]
I discovered [this](http://nomnoml.com/) the other day. Maybe consider switching to it instead of versioning images? It would make contributing graphs easier.

**Comment by regularjack** [comment]
Thanks @greg0ire, nomnoml looks interesting, I'll consider it when I address this issue.

**Comment by greg0ire** [comment]
Great! Here is the code if you decide to use that : 

```
[<start>st]->[<state>new]
[new]->[<state>expired]
[expired]->[<end>end]
[new]->[<state>approving]
[approving]->[<state>canceled]
[canceled]->[end]
[approving]->[<state>approved]
[approved]->[<state>depositing]
[depositing]->[<state>deposited]
[depositing]->[<state>failed]
[failed]->[end]
[deposited]->[end]
```

**Comment by regularjack** [comment]
FYI, I decided to go with graphviz, which was the easiest way to do it. You can take look at the fixed schemas at http://jmspaymentcorebundle.readthedocs.io/en/latest/model.html

**Comment by greg0ire** [comment]
Congrats, it looks nice! And you also get a textual representation, which is really good too!

**Comment by regularjack** [comment]
Fixed by #181
